src/components/cards/LogoutCard.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect } from 'react';
+import useToast from '../../hooks/useToast';
+import { Logout, resetLogoutState } from '../../redux/features/logoutSlice';
+import { useAppDispatch, useAppSelector } from '../../redux/hooks/hooks';
+import { ButtonIcon } from '../buttons/ButtonIcon';
+import IconLoader from '../Loaders/IconLoader';
+
+const LogoutCard = () => {
+	const dispatch = useAppDispatch();
+	const { isLoading } = useAppSelector((state) => state.logout);
+	const { showErrorMessage, showWorningMessage } = useToast();
+
+	const handleLogout = () => {
+		try {
+			dispatch(Logout());
+			showWorningMessage("You've logged out!");
+		} catch (error) {
```

Comment:
same case here, let's get message from backend response not hardcoded one

Reply:
Done